src/components/Plants/index.tsx

```diff
@@ -1,17 +1,55 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { ServerOrganization } from 'src/types/Organization';
 import strings from 'src/strings';
 import TfMain from 'src/components/common/TfMain';
-import Title from 'src/components/common/Title';
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { Select } from '@terraware/web-components';
+import { listPlantingSites } from 'src/api/tracking/tracking';
+import { PlantingSite } from 'src/api/types/tracking';
+import { useDeviceInfo } from '@terraware/web-components/utils';
 
 type PlantsDashboardProps = {
   organization: ServerOrganization;
 };
 
 export default function PlantsDashboard(props: PlantsDashboardProps): JSX.Element {
+  const { organization } = props;
+  const [selectedPlantingSite, setSelectedPlantingSite] = useState<PlantingSite>();
+  const [plantingSites, setPlantingSites] = useState<PlantingSite[]>([]);
+  const theme = useTheme();
+  const { isMobile } = useDeviceInfo();
+
+  useEffect(() => {
+    const populatePlantingSites = async () => {
+      const serverResponse = await listPlantingSites(organization.id);
+      if (serverResponse.requestSucceeded) {
+        setPlantingSites(serverResponse.sites ?? []);
+      }
+    };
+    populatePlantingSites();
+  }, [organization]);
+
+  const onChangePlantingSite = (newValue: string) => {
+    setSelectedPlantingSite(plantingSites.find((ps) => ps.name === newValue));
```

Comment:
ideally set it in the url path, have another useEffect that runs off the url path param to set the selected planting site